### DISCUSSION TITLE
Welcome to Term.jl Discussions!

### DISCUSSION THREAD

**Discussion by FedeClaudi** [FedeClaudi/Term.jl#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by snipes2** [comment]
Is there a way to have Panel just change the border without deleting the contents? I'm using it for a menu and when the menu is no longer active I want to change the border to HEAVY_EDGE but not delete the contents so the options still show

**Comment by asmarzec** [comment]
SimpleMenu with exactly two items crashes.

Reason:
vstack(::String, ::String) is dispatched instead of
vstack(::RenderablesUnion...),
which returns String instead of Renderable.

**Comment by asmarzec** [comment]
In refresh!():

- nlines_prev > i
+ nlines_prev >= i

Description:

The last line is never compared with the previous frame, 
causing unnecessary redraws and, under some circumstances, terminal rendering artifacts.
